### DISCUSSION TITLE
How to get DA proof by data bytes?

### DISCUSSION THREAD

**Discussion by fjchen7** [availproject/avail#584] [Q&A] [answered]
I have checked the [example](https://github.com/availproject/avail/blob/89b7fc7406dcb1146c59ab9e47901408dc8240fb/avail-subxt/examples/vector_send_msg.rs#L116-L150), and it seems that I can only query data proof by `tx_index` and `block_hash`.

Suppose I only have the data bytes, which has been already submitted to avail. How can I get the proof by data bytes? Very appreciate if someone provides an example.

**Comment by robin-rrt** [comment, marked as answer]
You cannot query the data proof without the blockhash.
